Userland/Applications/Screenshot/Screenshot.gml

```diff
@@ -36,28 +36,45 @@
             }
         }
 
-        @GUI::Widget {
+        @GUI::GroupBox {
+            title: "Output:"
             layout: @GUI::VerticalBoxLayout {
-                margins: [0, 0, 0, 8]
+                margins: [4]
+                spacing: 0
             }
 
-            @GUI::CheckBox {
+            @GUI::RadioButton {
+                name: "output_to_clipboard"
+                text: "Clipboard"
+                checked: true
```

Comment:
Maybe file output should be the default?

(not sure; up to you to make decision)

Reply:
Done.